src/warc2zim/converter.py

```diff
@@ -130,6 +130,7 @@ def __init__(self, args):
 
         self.indexed_urls = set({})
         self.revisits = {}
+        self.existing_entries = set({})
```

Comment:
I think it's more of a `warc_urls` than `existing_entries` which is unclear about whether processed or not and whether thoses are ZIM entries or WARC entries and given it's only URLs, entries itself is misleading

Reply:
Renamed to `warc_urls`